src/Microsoft.NET.Test.Sdk_props

```diff
@@ -11,6 +11,9 @@ Copyright (c) .NET Foundation. All rights reserved.
 -->
 
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
```

Comment:
I believe this props file is only dropped into the netcoreapp folder of the package - so if the unit test is targeting net46, it won't have this capability. There's a different issue for which I think there should just be one props file - so maybe this is fine if that issue is fixed.